stories/MemoryRouter.js

```diff
@@ -0,0 +1,18 @@
+export class MemoryRouter {
+  constructor(initialState = {}) {
+    this._memoryState = initialState;
+  }
+  write(routeState) {
+    this._memoryState = routeState;
+  }
+  read() {
+    return this._memoryState;
+  }
+  createURL() {
+    return '';
+  }
+  onUpdate() {
+    return {};
+  }
+  dispose() {}
```

Comment:
The state persists between the stories. That could be unexpected while we test, worth reset to the initial state.